app/src/main/java/org/mifos/mobile/ui/fragments/AboutUsFragment.kt

```diff
@@ -0,0 +1,60 @@
+package org.mifos.mobile.ui.fragments
+
+import android.content.Intent
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.TextView
+import butterknife.BindView
+import butterknife.ButterKnife
+import butterknife.OnClick
+import com.google.android.gms.oss.licenses.OssLicensesMenuActivity
+import org.mifos.mobile.BuildConfig
+import org.mifos.mobile.R
+import org.mifos.mobile.ui.activities.PrivacyPolicyActivity
+import org.mifos.mobile.ui.fragments.base.BaseFragment
+import java.util.*
+
+/*
+~This project is licensed under the open source MPL V2.
+~See https://github.com/openMF/self-service-app/blob/master/LICENSE.md
+*/   class AboutUsFragment : BaseFragment() {
+    @kotlin.jvm.JvmField
+    @BindView(R.id.tv_app_version)
+    var tvAppVersion: TextView? = null
+
+    @kotlin.jvm.JvmField
+    @BindView(R.id.tv_copy_right)
+    var tvCopyRight: TextView? = null
+    var rootView: View? = null
+    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?,
+                              savedInstanceState: Bundle?): View? {
+        rootView = inflater.inflate(R.layout.fragment_about_us, container, false)
+        ButterKnife.bind(this, rootView!!)
+        setToolbarTitle(getString(R.string.about_us))
+        tvAppVersion!!.text = getString(R.string.app_version, BuildConfig.VERSION_NAME)
```

Comment:
I think it is much better to use the safe call `?.` (One of the biggest advantages of using Kotlin) instead of `!!`, because it leads the App to crash if our view or any other item is null or it has not been initiated. You can replace all `!!` to `?.` in a file, simply by entering `Ctlr+r`.

Reply:
Thank you for your insights @jawidMuhammadi. I guess I had failed to notice a lot of not-null assertion operators since this conversion was straight from Java. Though there are quite a few functions in the app which requires a non-null object (For instance, `ButterKnife.bind()` is a Java function and doesn't support null safety yet), I would check for usages where safe call is possible and update it soon. Thanks again :)